src/components/Multiselect/__tests__/Multiselect.spec.js

```diff
@@ -235,29 +235,30 @@ describe('Multiselect', () => {
 
   describe('deselecting an option', () => {
 
-    it('removes the badge from the input', async () => {
+    it('emits a update:modelValue event', async () => {
       const props = { ...initialProps, modelValue: [{ label: 'United States', value: 'US' }] };
-      const { findByText, findAllByText } = renderComponent({ props });
+      const { getAllByText, getByRole, emitted } = renderComponent({ props });
 
-      const beforeText = await findAllByText('United States');
+      const beforeText = getAllByText('United States');
       expect(beforeText).toHaveLength(1); // in the badge
 
-      const button = await findByText(en.multiselect.deselectLabel);
-      userEvent.click(button);
+      const button = getByRole('button', { name: en.multiselect.deselectLabel });
+      await userEvent.click(button);
 
-      const afterText = await findAllByText('United States');
-      expect(afterText).toHaveLength(1); // in the options list
+      const emittedEvent = emitted();
+      expect(emittedEvent).toHaveProperty('update:modelValue');
+      expect(emittedEvent['update:modelValue'][0][0]).toHaveLength(0);
```

Comment:
I changed what this test is checking for because when testing `Multiselect` it doesn't actually remove the badge from the input when you deselect a value.  It emits the update but I'm guessing it's expecting its parent to change the `modelValue` which would then remove the badge from the input, but the `Multiselect` doesn't seem to do that by itself.  I'm not sure how this ever worked 🤷‍♀️ .